templates/yelp_polarity/templates.yaml

```diff
@@ -0,0 +1,67 @@
+dataset: yelp_polarity
+templates:
+  01dc166f-0774-4be0-b606-2beb2252d9b5: !Template
+    id: 01dc166f-0774-4be0-b606-2beb2252d9b5
+    jinja: '{{ text }} Overall, the experience is ||| {{ ["good", "bad"][label] }}.'
```

Comment:
can you remove these comma at the end? it will make the eval more clean (log-prob of the whole label sequence)